cmd/utils/dev.go

```diff
@@ -50,11 +52,37 @@ func LoadDev(devPath, namespace, k8sContext string) (*model.Dev, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	loadDevRc(dev)
 	loadContext(dev, k8sContext)
 	loadNamespace(dev, namespace)
 	return dev, nil
 }
 
+func loadDevRc(dev *model.Dev) {
+	defaultDevRcPath := filepath.Join(config.GetOktetoHome(), ".okteto.yml")
+	secondaryDevRcPath := filepath.Join(config.GetOktetoHome(), ".okteto.yaml")
```

Comment:
can we call it `okteto.yaml` without the dot, since this is a file that's meant to be edited by the user?